src/assets/style/landmark/_stripView.less

```diff
@@ -18,18 +18,19 @@
     right: -390px;
 }
 
+.stripTitle {
+    color: #ffffff;
+    font-size: 2em;
+    background-color: #222222;
+    text-align: center;
+}
+
 .stripView-list {
-    max-height: 100%;
     // LESS itself tries to evaluate calc(), using ~"" here to prevent that
     width: ~"calc(100% - " @spacing-2x ~")";
-    position: absolute;
-    top: @spacing-2x;
-    bottom: @spacing-2x;
-    left: @spacing-2x;
-    display: flex;
-    flex-direction: column-reverse;
```

Comment:
Why did you remove the `column-reverse`?  The idea here was to have the list populate from the bottom up, just like a real flight strip list.  By removing that, each list will populate from the top down.

@erikquinn @Fechulo are you guys of with that change?  It's a slight departure from realism, though probably makes things a little easier to manage code-wise.

Reply:
I reverted this back to ``column-reverse``